test/Mvc.Analyzers.Test/TestFiles/TopLevelParameterNameAnalyzerTest/DiagnosticsAreReturned_ForControllerActionsWithParametersThatMatchProperties.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace Microsoft.AspNetCore.Mvc.Analyzers.TopLevelParameterNameAnalyzerTestFiles
```

Comment:
I would just double check that none of these test classes result in long path issues.

Reply:
ok. I know the perf lab had some issues with long paths that @BrennanConroy managed to evaporate.

Reply:
If it isn't product code or benchmarks code then the perf runs will be fine